src/Results/Results.js

```diff
@@ -2,16 +2,27 @@ import React, { Component } from 'react'
 
 class Results extends Component {
 	render() {
-		const {question, alternatives, votes} = this.props
+		const { question, alternatives, votes, open, users } = this.props
+
+		const numVotes = votes.reduce((a, b) => a + b, 0);
+		// user.attending is a boolean
+		// However, true + true = 2 (and so on) in Javascript, so the map is actually unnecessary
+		const attending = users.map(u => u.attending ? 1 : 0).reduce((a, b) => a + b, 0)
+
 		return (
 			<div>
 				<h1>{ question }</h1>
+				<h3>Antal som röstat: <span style={{color: "black"}}>{numVotes} / {attending}</span></h3>
 	            <ul className="results">
 	              { alternatives.map((alternative, index) => (
 	                <li key={JSON.stringify(alternative)}>
-	                  <div className="theme-color bar-label">{ alternative.content } ({ votes[index] })</div>
+	                  <div className="theme-color bar-label">{ alternative.content } { open ? "?" : votes[index] }</div>
```

Comment:
Ingen kanske nämnt det för dig. Meeen om du har ett långt svarsalternativ så kunde man tidigare inte se röstsiffrorna. 
Skulle vara najs om det går att åtgärda.

Reply:
Om du inte redan gjorde det.

Reply:
Fixed in b4ecea8